src/agent/pythd/api/rpc.rs

```diff
@@ -477,792 +499,3 @@ async fn serve(
 
     tokio::task::spawn(serve).await.map_err(|e| e.into())
 }
-
-#[cfg(test)]
```

Comment:
Actually there is some value to these, I will bring these back but in a slightly different form.

Reply:
i think the adaptor tests are a bit useless but this is good.